### PR TITLE
Camera: Set up Python project

### PR DESCRIPTION
**Pull request by kantoniak**
* Add setup scripts
* Create starting Python script
* Add workflow to force Python linting and conventions for camera

### CODE REVIEW

File: camera/rpi-files/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ignore Python environment
+env
+
+# Ignore app logs
+app.log
```
Comment (avan1235) [inline camera/rpi-files/.gitignore]: maybe move to main gitignore and remove this?

File: camera/rpi-files/setup-development.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+KEY_FILENAME="github-camera-dev-key"
+
+echo "Setting up git..."
+read -p "Git user.name: " git_username
+read -p "GitHub email address: " gh_email
+git config --global user.name "${git_username}"
+git config --global user.email "${gh_email}"
```
Comment (avan1235) [inline camera/rpi-files/setup-development.sh]: maybe keep single convention git/gh

File: camera/rpi-files/setup.sh [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+sudo apt-get install python3.7
+
+# Set up virtual environment
+cd rpi-files
+python3 -m venv env
+
+# Install requirements
+source env/bin/activate
+pip install -r requirements.txt
+deactivate
```
Comment (avan1235) [inline camera/rpi-files/setup.sh]: let's thisnk of using && again because there can definitely be errors

File: camera/rpi-files/.gitignore (hunk 1 above)
Comment (kantoniak) [reply camera/rpi-files/.gitignore]: I'd rather keep .gitignores separate in different projects. I will move up to camera.

Comment (kantoniak) [reply camera/rpi-files/.gitignore]: Actually, I'd rather keep it as it. `rpi-files` is a separate filesystem for Raspbian, we may ship only this folder to rpi later.

File: camera/rpi-files/setup.sh (hunk 3 above)
Comment (kantoniak) [reply camera/rpi-files/setup.sh]: Done.

File: camera/rpi-files/setup-development.sh (hunk 2 above)
Comment (kantoniak) [reply camera/rpi-files/setup-development.sh]: Done

**Pull request merged**